_dev/src/components/smart-shopping-campaign/smart-shoppinh-campaign-table-list.spec.ts

```diff
@@ -165,6 +165,6 @@ describe('smart-shopping-campaign-table-list.vue - show banner SSC ads if user h
   });
   it('should display buttons', () => {
     expect(wrapper.find('[data-test-id="redirect-to-reporting-button"]').exists()).toBeTruthy();
-    expect(wrapper.find('[data-test-id="create-campaign-button"]').exists()).toBeTruthy();
+    expect(wrapper.find('[data-test-id="create-campaign-button"]').exists()).toBeFalsy();
```

Comment:
This change the string after `it()`, we don't anymore that all buttons are being displayed